packages/web-components/src/components/va-table/va-table-inner/va-table-slot.css

```diff
@@ -14,8 +14,6 @@ va-table > va-table-row[slot] button svg {
   width: 1.5rem;
 }
 
-@media screen and (min-width: 768px) {
-  va-table-row span.text-align-right {
-    text-align: right!important;
-  }
+va-table-row span.text-align-right {
```

Comment:
Can you elaborate on why the media query isn't needed anymore?

Reply:
Took it out while I was developing, it made it so only larger screen sizes could be right-aligned. I'll put it back just in case there are some teams relying on it for some reason.